[CS2103-T14-4] Personal Expense Manager

choonx99 opened the pull request:
@muhammadaljunied
@Cary-Xx
@limxuanhui

Cary-Xx committed:
Fix Json

choonx99 committed:
refactored test model files and minor updates to test data files

choonx99 committed:
refactored test logic files and minor refactors to main java files

choonx99 committed:
fix checkstyle

choonx99 committed:
fix checkstyle

choonx99 committed:
fix checkstyle

choonx99 committed:
fix test case errors

choonx99 committed:
fix checkstyle

choonx99 committed:
Merge branch 'Refactor_ZH' of https://github.com/AY1920S1-CS2103-T14-4/main into Refactor_CW

choonx99 committed:
commented failed date TCs

Cary-Xx committed:
Fix format for Date

choonx99 committed:
fix test case error

choonx99 committed:
fix tc error

Cary-Xx committed:
Merge pull request #25 from choonx99/Refactor_CW

Refactor to MYMorise Test

Cary-Xx committed:
Merge branch 'Refactor_ZH' of github.com:AY1920S1-CS2103-T14-4/main into Re-back

Cary-Xx committed:
Fix test and Update date and name regex

Cary-Xx committed:
Make date optional and fix minor issues

Cary-Xx committed:
Minor fix

Cary-Xx committed:
Add test

Cary-Xx committed:
Update doc

Cary-Xx committed:
Merge pull request #24 from AY1920S1-CS2103-T14-4/Refactor_ZH

Refactor to MYMorise

choonx99 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T14-4/main

choonx99 committed:
Update app name, icon and userguide link (#62)

Cary-Xx committed:
Uniform expense format

Cary-Xx committed:
Merge pull request #63 from Cary-Xx/develop_ZH

Uniform expense format

muhammadaljunied committed:
Add Currency Class

choonx99 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T14-4/main

muhammadaljunied committed:
Added Currency to Expense

muhammadaljunied committed:
Resolve Test failure

muhammadaljunied committed:
Resolve Checkstyle

Cary-Xx committed:
Add autocomplete for user input

muhammadaljunied committed:
Resolve CheckstyleTest

muhammadaljunied committed:
Merge pull request #64 from muhammadaljunied/add-currency-ISO

Added Currency to Expense

Cary-Xx committed:
Update test and minor fix (#67)

* Update test and minor fix

* Fix

choonx99 committed:
added add budget function, checks to prevent budget period clashes, expenses to auto include into budget if date falls into budget period

Cary-Xx committed:
Fix reading logic and caret selection issue

choonx99 committed:
Fixed test cases for add budget

choonx99 committed:
fixed expenselist storage inside budget and checkstyle errors

Cary-Xx committed:
Arrange

choonx99 committed:
fix checkstyle error

choonx99 committed:
fixed checkstyle error

choonx99 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T14-4/main

choonx99 committed:
update Userguide

Cary-Xx committed:
Merge pull request #71 from choonx99/Add_Budget

Add budget

choonx99 committed:
update Developerguide

Cary-Xx committed:
Update developer guide

choonx99 committed:
update developer guide diagrams

Cary-Xx committed:
Merge pull request #75 from AY1920S1-CS2103-T14-4/Add_Budget

Add budget V1

Cary-Xx committed:
Merge pull request #76 from AY1920S1-CS2103-T14-4/UpdateDocs_CW

Update docs

Cary-Xx committed:
Prepare for release v1.2.1

muhammadaljunied committed:
Added JSON Parsing for exchange rate file.

muhammadaljunied committed:
Comment out failing test cases.

muhammadaljunied committed:
Update DG and PUML

Cary-Xx committed:
Fix

Cary-Xx committed:
Merge pull request #72 from AY1920S1-CS2103-T14-4/Autocomplete_ZH

AutoComplete V1

Cary-Xx committed:
Fix

Cary-Xx committed:
Revert "Fix"

muhammadaljunied committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T14-4/main into currency-update

* 'master' of https://github.com/AY1920S1-CS2103-T14-4/main:
  Revert "Fix"
  Fix
  Fix
  Prepare for release v1.2.1
  Update developer guide
  update developer guide diagrams
  update Developerguide
  update Userguide
  fixed checkstyle error
  fix checkstyle error
  Arrange
  fixed expenselist storage inside budget and checkstyle errors
  Fixed test cases for add budget
  Fix reading logic and caret selection issue
  Add autocomplete for user input
  added add budget function, checks to prevent budget period clashes, expenses to auto include into budget if date falls into budget period
  revert removeAddr merge
  Remove trailing white space from contactus doc
  safe delete Address class

# Conflicts:
#	docs/DeveloperGuide.adoc
#	src/main/java/seedu/address/MainApp.java
#	src/main/java/seedu/address/model/Model.java
#	src/main/java/seedu/address/model/ModelManager.java
#	src/main/java/seedu/address/model/UserPrefs.java
#	src/main/java/seedu/address/storage/Storage.java
#	src/main/java/seedu/address/storage/StorageManager.java
#	src/test/java/seedu/address/logic/LogicManagerTest.java
#	src/test/java/seedu/address/logic/commands/AddCommandIntegrationTest.java
#	src/test/java/seedu/address/logic/commands/AddCommandTest.java
#	src/test/java/seedu/address/logic/commands/ClearCommandTest.java
#	src/test/java/seedu/address/logic/commands/DeleteCommandTest.java
#	src/test/java/seedu/address/logic/commands/EditCommandTest.java
#	src/test/java/seedu/address/logic/commands/FindCommandTest.java
#	src/test/java/seedu/address/logic/commands/ListCommandTest.java
#	src/test/java/seedu/address/model/ModelManagerTest.java
#	src/test/java/seedu/address/storage/StorageManagerTest.java

muhammadaljunied committed:
Resolve checkstyle issues on main and test

muhammadaljunied committed:
comment out failing test

muhammadaljunied committed:
resolve checkstyle

jessicax941 commented on docs/DeveloperGuide.adoc:
```diff
@@ -235,12 +236,41 @@ image::CommitActivityDiagram.png[]
+
+The model is constructed through `java.seedu.address.model.autocomplete` which contains `AutoCompleteModel` and `Word`.
+
+The Ui part is implemented through `java.seedu.address.ui.QueryCard`on top of `CommandBox`.
```
I think the missing space between "QueryCard`" and "on" causes this part of the sentence to be wrongly formatted in the DG.

jessicax941 commented on docs/DeveloperGuide.adoc:
```diff
@@ -235,12 +236,41 @@ image::CommitActivityDiagram.png[]
+`AutoComplete#getSuggestions()` is invoked.
+
+Step 3. `initAc()` calls `AutoComplete#readWordsFromFile()` which reads the vocabulary from our local dictionary to get
+the "database and then construct an `AutocompleteModel` with the vocabulary read"
```
Not sure what the inverted commas are there for.

jessicax941 commented on docs/DeveloperGuide.adoc:
```diff
@@ -235,12 +236,41 @@ image::CommitActivityDiagram.png[]
+
+The following sequence diagram shows how autocomplete operation works:
+
+image::AutocompleteSequenceDiagram.png[]
```
Maybe the group could revise this sequence diagram as it is unclear (e.g. why is userInput on an arrow pointing to :CommandBox? Shouldn't the arrows only be for method calls?) and it is not coherent with the description above the diagram.
I think the return arrows should also be dotted.

jessicax941 commented on docs/DeveloperGuide.adoc:
```diff
@@ -235,12 +236,41 @@ image::CommitActivityDiagram.png[]
+Given below is an example usage scenario and how the autocomplete mechanism behaves at each step. (p.s. details are
+omitted)
+
+Step 1. The user launch MYMorise and the user will be prompted to enter a command as shown in the command box.
```
Quite detailed description.

Cary-Xx committed:
Fix autocomplete bug and date invalidation

Cary-Xx committed:
Fix test

Cary-Xx committed:
Fix

choonx99 committed:
added currency to budget

choonx99 committed:
added currency to budget

choonx99 committed:
updated delete and edit for budget, view budget list,

choonx99 committed:
updated delete and edit for budget, view budget list,

Cary-Xx committed:
Fix expenseList and budgetList display issue

Cary-Xx committed:
Fix expenseList and budgetList display issue

choonx99 committed:
fixed test case errors

choonx99 committed:
fixed test case errors

choonx99 committed:
update ug

Cary-Xx committed:
Fix

Cary-Xx committed:
Fix

Cary-Xx committed:
Merge pull request #80 from AY1920S1-CS2103-T14-4/UpdateDocs_CW

update ug

Cary-Xx committed:
Merge pull request #79 from Cary-Xx/FixRelease

Fix autocomplete bug and date invalidation

choonx99 committed:
update delete hack

choonx99 committed:
update delete hack

Cary-Xx committed:
update ug

Cary-Xx committed:
Update Ui.png

Cary-Xx committed:
Merge pull request #81 from Cary-Xx/FixRelease

update ug

muhammadaljunied committed:
HTTP Client

muhammadaljunied committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T14-4/main into currency-update

choonx99 committed:
update user stories for budget in DG

muhammadaljunied committed:
Updated UG for Currency Conversion

Cary-Xx committed:
Fix release

Cary-Xx committed:
Merge pull request #82 from AY1920S1-CS2103-T14-4/UpdateDocs_CW

update user stories for budget in DG

Cary-Xx committed:
Fix DG

muhammadaljunied committed:
Update UG & DG

muhammadaljunied committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T14-4/main into Week-11

muhammadaljunied committed:
Todos

Cary-Xx committed:
Merge pull request #84 from muhammadaljunied/Week-11

Updated Currency UG and DG

Cary-Xx committed:
Merge pull request #83 from Cary-Xx/FixRelease

Fix release

muhammadaljunied committed:
Undo test cases comment

Cary-Xx committed:
Fix budget and update UI

Cary-Xx committed:
Fix budget and update UI

choonx99 committed:
fixed delete and edit functions

choonx99 committed:
fixed delete and edit functions

Cary-Xx committed:
Merge pull request #87 from Cary-Xx/view

Fix budget and update UI

choonx99 committed:
fix amount to remove currency prefix and convert to 2 decimals

choonx99 committed:
fix amount to remove currency prefix and convert to 2 decimals

choonx99 committed:
update editBudget to change state after edit

choonx99 committed:
update editBudget to change state after edit

choonx99 committed:
Update UG for Practical dry run

choonx99 committed:
Merge branch 'Add_Budget' of https://github.com/AY1920S1-CS2103-T14-4/main into Add_Budget

Cary-Xx committed:
Fix for PE v1.3

Cary-Xx committed:
Fix

Cary-Xx committed:
Merge pull request #88 from AY1920S1-CS2103-T14-4/UpdateDocs_CW

Update UG for Practical dry run

Cary-Xx committed:
Update Ui.png

Cary-Xx committed:
Fix tests

Cary-Xx committed:
Fix

Cary-Xx committed:
Fix UI

Cary-Xx committed:
Update UI

choonx99 committed:
update UG

choonx99 committed:
update UG

choonx99 committed:
Merge branch 'Add_Budget' of https://github.com/AY1920S1-CS2103-T14-4/main into Add_Budget

# Conflicts:
#	src/main/java/seedu/address/logic/Logic.java
#	src/main/java/seedu/address/logic/LogicManager.java
#	src/main/java/seedu/address/logic/commands/AddBudgetCommand.java
#	src/main/java/seedu/address/logic/commands/ClearCommand.java
#	src/main/java/seedu/address/logic/commands/CommandResult.java
#	src/main/java/seedu/address/logic/commands/DeleteCommand.java
#	src/main/java/seedu/address/logic/commands/EditBudgetCommand.java
#	src/main/java/seedu/address/logic/commands/EditCommand.java
#	src/main/java/seedu/address/logic/commands/ViewBudgetCommand.java
#	src/main/java/seedu/address/logic/commands/ViewBudgetListCommand.java
#	src/main/java/seedu/address/model/Model.java
#	src/main/java/seedu/address/model/ModelManager.java
#	src/main/java/seedu/address/model/budget/Budget.java
#	src/main/java/seedu/address/model/expense/Amount.java
#	src/main/java/seedu/address/model/expense/Expense.java
#	src/main/java/seedu/address/ui/MainWindow.java
#	src/main/java/seedu/address/ui/StatsDisplay.java
#	src/main/resources/view/MainWindow.fxml
#	src/test/java/seedu/address/logic/commands/AddCommandTest.java
#	src/test/java/seedu/address/testutil/BudgetBuilder.java

choonx99 committed:
GUI fix

choonx99 committed:
fix prevent amount 0, added extra check to prevent overlapping budget periods, update edit command to show correct error message

choonx99 committed:
convert expenselist name to defaultexpenselist in modelmanager only

choonx99 committed:
converted viewstate to use enumeration. fixed list command and other commands to display centre panel properly

muhammadaljunied committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T14-4/main into currency-update

# Conflicts:
#	docs/UserGuide.adoc

muhammadaljunied committed:
Added automatic conversion of Foreign Currency

muhammadaljunied committed:
-Fixed broken test-cases
-Added new logic for currency conversion
-Added Singleton to expose conversion data

muhammadaljunied committed:
Merge branch 'Add_Budget' of https://github.com/AY1920S1-CS2103-T14-4/main into currency-update

# Conflicts:
#	src/main/java/seedu/address/model/ModelManager.java
#	src/main/java/seedu/address/model/expense/Amount.java
#	src/main/java/seedu/address/model/expense/Currency.java
#	src/test/java/seedu/address/logic/commands/AddCommandTest.java
#	src/test/java/seedu/address/logic/commands/EditCommandTest.java

muhammadaljunied committed:
Checkstyle and CheckstyleTest correction

muhammadaljunied committed:
Trigger check's rerun

muhammadaljunied committed:
Trigger Checks again

muhammadaljunied committed:
Trigger checks again

muhammadaljunied committed:
Merge pull request #77 from muhammadaljunied/currency-update

Currency update - Added JSON Support for Currency

Cary-Xx committed:
PPP draft

Cary-Xx committed:
Fix

Cary-Xx committed:
Merge pull request #132 from Cary-Xx/week12

ziheng PPP draft

choonx99 committed:
added test cases for budget

choonx99 committed:
update ppp

muhammadaljunied committed:
PPP Draft

choonx99 committed:
update ppp

muhammadaljunied committed:
Correct some typos and checkstyle

choonx99 committed:
Update docs cw (#130)

* update UG

* update UG

* update ppp

* update ppp

muhammadaljunied committed:
Merge branch 'master' into Week-12-Docs

muhammadaljunied committed:
Merge pull request #133 from muhammadaljunied/Week-12-Docs

PPP Draft

Cary-Xx committed:
Add history command and minor fix

Cary-Xx committed:
test

choonx99 committed:
update to Addcommand and mainwindow.fxml

choonx99 committed:
added extra test cases

muhammadaljunied committed:
Fixed crash from issue #135. Now sample data includes all supported currencies

muhammadaljunied committed:
Checkstyle Fix

muhammadaljunied committed:
Added Exchange data is now loaded asynchronously from the endpoint

choonx99 committed:
standardized commands and added more test cases

muhammadaljunied committed:
Fixed Regression for conversions in Budgets where Budget currency is not baseCurrency (SGD).
Currency Rates are now persisted to disk
Refactored Currency property name from value -> name
Shifted pulling of rate further up the chain to where the Currency constructor is called (because thats where it needs the data).
Checked for Regressions
Testcases updated
Checkstyle.

choonx99 committed:
changed data to 12hr format

choonx99 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T14-4/main into UpdateDocs_CW

# Conflicts:
#	docs/team/overview.adoc

choonx99 committed:
updated standardized commands and last day TIP in UG

choonx99 committed:
updated negative amount left to be red

muhammadaljunied committed:
Added Dynamic titles to the UI

Cary-Xx committed:
Fix merge conflict and checkstyle

Cary-Xx committed:
dummy

Cary-Xx committed:
Retest

Cary-Xx committed:
Test

Cary-Xx committed:
Merge pull request #140 from Cary-Xx/final

Add history command and minor fix

Cary-Xx committed:
Fix merge conflict

Cary-Xx committed:
Fix

Cary-Xx committed:
Fix

Cary-Xx committed:
Merge pull request #143 from AY1920S1-CS2103-T14-4/Add_Budget

added some test cases

Cary-Xx committed:
Merge pull request #144 from muhammadaljunied/Issue-135-Fix

Fixed crash from issue #135.

Cary-Xx committed:
Merge branch 'master' into ExchangeData-HTTPSupport

Cary-Xx committed:
Merge pull request #145 from muhammadaljunied/ExchangeData-HTTPSupport

Exchange data http support

Cary-Xx committed:
Merge branch 'master' into UI-DynamicTitles

Cary-Xx committed:
Merge pull request #147 from muhammadaljunied/UI-DynamicTitles

UI dynamic titles

Cary-Xx committed:
Merge pull request #146 from AY1920S1-CS2103-T14-4/UpdateDocs_CW

Update docs cw

Cary-Xx committed:
Fix

Cary-Xx committed:
Merge pull request #148 from Cary-Xx/Final

Fix

muhammadaljunied committed:
Completed refactor of Tags (Plural) -> Tag (Singular)
-Other fixes to Currency not processing correctly.
-Refactored test cases to match and removed those that were no longer relevant.

muhammadaljunied committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T14-4/main into Model-RemoveMultiTags

# Conflicts:
#	src/main/java/seedu/address/logic/parser/AddExpenseCommandParser.java
#	src/main/java/seedu/address/model/tag/Tag.java
#	src/test/java/seedu/address/logic/LogicManagerTest.java
#	src/test/java/seedu/address/logic/commands/CommandTestUtil.java
#	src/test/java/seedu/address/logic/commands/EditExpenseCommandTest.java
#	src/test/java/seedu/address/logic/parser/AddExpenseCommandParserTest.java
#	src/test/java/seedu/address/logic/parser/EditExpenseCommandParserTest.java
#	src/test/java/seedu/address/storage/JsonAdaptedExpenseTest.java
#	src/test/java/seedu/address/testutil/EditExpenseDescriptorBuilder.java
#	src/test/java/seedu/address/testutil/ExpenseUtil.java

choonx99 committed:
remove commented UC format

muhammadaljunied committed:
Merge pull request #149 from muhammadaljunied/Model-RemoveMultiTags

Completed refactor of Tags (Plural) -> Tag (Singular)

muhammadaljunied committed:
Fix a UI Bug with untagged expenses rendering an empty tag on expense card.

muhammadaljunied committed:
Fixed a UI related bug rendering empty tags for expenses which are untagged.

muhammadaljunied committed:
Bugfix on invalid currencies passing through the checks
Update Tag to capitalize first letter

muhammadaljunied committed:
Merge pull request #150 from muhammadaljunied/Model-RemoveMultiTags

Bugfixes for regression

choonx99 committed:
update piechart to show by tags, and update panel title to reflect properly on command

choonx99 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T14-4/main into New_Add_Budget

choonx99 committed:
minor edit to Stats display

choonx99 committed:
update some DG diagrams

choonx99 committed:
fix

choonx99 committed:
fix

choonx99 committed:
updated UI colors

muhammadaljunied committed:
Standardised converted amount and rates printing.

muhammadaljunied committed:
Merge pull request #152 from AY1920S1-CS2103-T14-4/UpdateDocs_CW

Update docs cw

muhammadaljunied committed:
Merge pull request #156 from muhammadaljunied/Bugfix-001

Standardised converted amount and rates printing.

Cary-Xx committed:
update history and fix bugs

Cary-Xx committed:
Revert sample

Cary-Xx committed:
Fix

Cary-Xx committed:
Merge pull request #151 from AY1920S1-CS2103-T14-4/New_Add_Budget

New add budget

choonx99 committed:
update sample data, shift uniquebudgetlist and budgetlist to model

choonx99 committed:
update sample data, shift uniquebudgetlist and budgetlist to model

choonx99 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T14-4/main into New_Add_Budget

Cary-Xx committed:
fix website

choonx99 committed:
update sampledata

choonx99 committed:
Merge branch 'New_Add_Budget' of https://github.com/AY1920S1-CS2103-T14-4/main into New_Add_Budget

choonx99 committed:
fix GUI

choonx99 committed:
fix checkstyle

Cary-Xx committed:
Draft UG

Cary-Xx committed:
Merge pull request #157 from AY1920S1-CS2103-T14-4/New_Add_Budget

New add budget

muhammadaljunied committed:
Updated testcases

muhammadaljunied committed:
Update Site headers

muhammadaljunied committed:
Merge branch 'master' into Bugfix-001

muhammadaljunied committed:
Merge pull request #158 from muhammadaljunied/Bugfix-001

Updated testcases

muhammadaljunied committed:
Updated build.gradle

muhammadaljunied committed:
Merge branch 'Bugfix-001' of https://github.com/muhammadaljunied/main into Bugfix-001

* 'Bugfix-001' of https://github.com/muhammadaljunied/main:
  Draft UG
  fix checkstyle
  fix GUI
  update sampledata
  fix website
  update sample data, shift uniquebudgetlist and budgetlist to model
  update sample data, shift uniquebudgetlist and budgetlist to model
  Fix
  Revert sample
  update history and fix bugs
  updated UI colors
  fix
  fix
  update some DG diagrams
  minor edit to Stats display
  update piechart to show by tags, and update panel title to reflect properly on command
  remove commented UC format

# Conflicts:
#	build.gradle

muhammadaljunied committed:
Merge pull request #159 from muhammadaljunied/Bugfix-001

Bugfix 001

muhammadaljunied committed:
Applied stashed changes

muhammadaljunied committed:
Refactored Amount Currency Date Nameand NameContainsKeywordsPredicate to Commons package

Cary-Xx committed:
Update ui

Cary-Xx committed:
Prepare for release

muhammadaljunied committed:
Refactored some files
Fixed a launch bug when folder doesn't exist

Cary-Xx committed:
Merge pull request #160 from Cary-Xx/finalDevelop

Update ui

muhammadaljunied committed:
Checkstyle

choonx99 committed:
updated diagrams

muhammadaljunied committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T14-4/main into Bugfix-001

choonx99 committed:
Merge branch 'New_Add_Budget' of https://github.com/AY1920S1-CS2103-T14-4/main into New_Add_Budget

muhammadaljunied committed:
Fixed Bug with Large inputs.

muhammadaljunied committed:
Merge pull request #161 from muhammadaljunied/Bugfix-001

Bugfix 001

Cary-Xx committed:
Update DG

Cary-Xx committed:
Update doc

Cary-Xx committed:
Merge pull request #162 from Cary-Xx/finalDevelop

Final develop

muhammadaljunied committed:
Fixed display bug

choonx99 committed:
update DG

Cary-Xx committed:
Fix

Cary-Xx committed:
Merge pull request #163 from muhammadaljunied/Bugfix-001

Fixed display bug

choonx99 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103-T14-4/main into New_Add_Budget

# Conflicts:
#	docs/DeveloperGuide.adoc

Cary-Xx committed:
Merge pull request #164 from AY1920S1-CS2103-T14-4/New_Add_Budget

Update Dg

Cary-Xx committed:
Merge branch 'master' of github.com:AY1920S1-CS2103-T14-4/main into finalDevelop

muhammadaljunied committed:
DG Update

muhammadaljunied committed:
Merge pull request #165 from muhammadaljunied/Bugfix-001

DG Update

choonx99 committed:
update ug

choonx99 committed:
Merge pull request #166 from AY1920S1-CS2103-T14-4/New_Add_Budget

update ug

muhammadaljunied committed:
PPP

muhammadaljunied committed:
Checkstyle

choonx99 committed:
update ppp

Cary-Xx committed:
Fix DG and PPP

Cary-Xx committed:
Merge branch 'master' of github.com:AY1920S1-CS2103-T14-4/main into finalDevelop

Cary-Xx committed:
Fix

Cary-Xx committed:
Fix

muhammadaljunied committed:
DG Sequence Diagram

Cary-Xx committed:
Merge pull request #168 from Cary-Xx/finalDevelop

Doc

choonx99 committed:
update PPP and add more pictures to UG

Cary-Xx committed:
Merge pull request #169 from AY1920S1-CS2103-T14-4/New_Add_Budget

Update PPP and UG

muhammadaljunied committed:
no message

Cary-Xx committed:
Merge pull request #167 from muhammadaljunied/Bugfix-001

PPP
